etc/helmfile-templates.yaml

```diff
@@ -0,0 +1,7 @@
+templates:
```

Comment:
would it be possible to also move the defaults from environments.yaml here? and then rename it to `etc/helmfile.yaml`?

Reply:
I played with these files a lot and couldn't find a more elegant solution, I don't remember if I tried what you're suggesting.